website/content/docs/connect/observability/access-logs.mdx

```diff
@@ -0,0 +1,202 @@
+---
+layout: docs
+page_title: Service Mesh Observability - Access Logs
+description: >-
+  Consul Service Mesh can emit logs for application connections and requests passing through mesh proxies. Learn how to configure access logs for your mesh.
+---
+
+# Access Logs
+
+Consul Service Mesh can emit logs for application connections and requests passing through mesh proxies, including gateways.
```

Comment:
nit: making the types of mesh proxies explicit (to ensure this isn't misread as "mesh gateways")
```suggestion
Consul Service Mesh can emit logs for application connections and requests passing through mesh proxies, including sidecars and gateways.
```